hoodie-hive/src/main/java/com/uber/hoodie/hive/HoodieHiveClient.java

```diff
@@ -556,7 +556,7 @@ public void close() {
   }
 
   @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
-  List<String> getPartitionsWrittenToSince(Optional<String> lastCommitTimeSynced) {
+  List<String> getPartitionsWrittenToSince(Option<String> lastCommitTimeSynced) {
```

Comment:
do we still need the `@suppresswarnings`?

Reply:
I think we should keep it there because we still use `Option`.